Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -283,6 +283,7 @@ public struct ConvertAction: Action, RecreatingContext {
     /// Converts each eligible file from the source documentation bundle,
     /// saves the results in the given output alongside the template files.
     mutating public func perform(logHandle: LogHandle) throws -> ActionResult {
+        let totalTimeMetric = benchmark(begin: Benchmark.Duration(id: "convert-total-time"))
```

Comment:
Should we remove the old "convert" benchmark? I think it's kind of confusing now to know what the difference between "convert-total-time" and "convert-processing" is? Unless you think the old value is still useful?

```
+-----------------------------------------------------------------------------------------------------+
| Metric                                   | Change     | Before               | After                |
+-----------------------------------------------------------------------------------------------------+
| Duration for 'bundle-registration' (sec) | +0.78%     | 2.95                 | 2.97                 |
| Duration for 'convert-processing' (sec)  | +20.22%    | 1.30                 | 1.56                 |
| Duration for 'convert-total-time' (sec)  | -32.89%    | 7.01                 | 4.70                 |
| Duration for 'finalize-navigation-index' | +2.82%     | 0.07                 | 0.07                 |
| Peak memory footprint (MB)               | -1.53%     | 207.48               | 204.30               |
| Data subdirectory size (MB)              | no change  | 57.40                | 57.40                |
| Index subdirectory size (MB)             | no change  | 0.47                 | 0.47                 |
| Total DocC archive size (MB)             | no change  | 75.69                | 75.69                |
| Topic Anchor Checksum                    | no change  | 65d4ff3050cd1106a21b | 65d4ff3050cd1106a21b |
| Topic Graph Checksum                     | no change  | 0a64c740a2ed5a246836 | 0a64c740a2ed5a246836 |
+-----------------------------------------------------------------------------------------------------+
```

Reply:
I think it's still useful to measure the time it takes to resolve links, construct auto-curation, and render all the content but we can use a different name for the metric. How about "processing-documentation" or just "processing"?

Reply:
Yeah that sounds good to me! And then I guess we just want to watch out for if "convert-total-time" ever grows much larger than "bundle-registration" + "documentation-processing"